packages/integrations/src/captureconsole.ts

```diff
@@ -29,51 +35,50 @@ export class CaptureConsole implements Integration {
   /**
    * @inheritDoc
    */
-  public setupOnce(_: (callback: EventProcessor) => void, getCurrentHub: () => Hub): void {
```

Comment:
Let's not change the signature of this. This integration is part of the public API.

Reply:
I think this should be fine for an API as any caller can still pass in anything that you passed into it previously (it accepts _more_, not _less_), but I can leave it as is for sure.

Reply:
If there's no reason to widen the API let's not do it.